### PR TITLE
Added labeler

### PR DESCRIPTION
**Pull request by mikolajhojda**

### CODE REVIEW

**Comment by logankilpatrick** [comment]
Hey @mikolajhojda lets look at an example: 
```
documentation: 
 - "**/*.md"
``` 

This is saying that there is a PR label that exists called "documentation" and any Pr with changes to a file path ending in a markdown file will trigger that label. 

Some of the newest examples you added don't make sense because the PR label is the same as the file path (the file path you put in doe snot exist). 

Also, some labels don't work for a pr labeler like "wont-fix". How can a specific fila path translate to the idea of "wont-fix"? I am not sure that will ever be possible. Hope this helps.

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+waiting for feedback: 
+- "waiting for feedback/*"
+
+waiting on author: 
+- "waiting on author/*"
+
+websockets: 
+- "websockets/*"
+
+wontfix:
```
Comment (logankilpatrick) [inline .github/labeler.yml]: delete this

**Comment by mikolajhojda** [comment]
Could the "test" label be added?

**Comment by logankilpatrick** [comment]
Yes, open an issue and ask for it!

**Comment by mikolajhojda** [comment]
https://github.com/JuliaWeb/HTTP.jl/issues/498

Added

**Comment by quinnj** [comment]
I'm not really familiar w/ what this yaml file does; auto adds labels to issues/PRs? @logankilpatrick , does this look correct to you now?

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+📖 documentation:
```
Comment (logankilpatrick) [inline .github/labeler.yml]: @quinnj this line is the only one that needs to change unless you have a "📖 documentation" label

Comment (logankilpatrick) [reply .github/labeler.yml]: oh you do, nvm!

**Comment by logankilpatrick** [comment]
This should be all good!

**Pull request merged**